steps/kobiton-app-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,91 @@
+title: Kobiton App Upload
```

Comment:
Please start the title with the action like `Upload application to Kobiton`

Reply:
Yes, sounds good